rocker-compiler/src/test/resources/rocker/parser/SwitchBlockWithDefault.rocker.html

```diff
@@ -0,0 +1,18 @@
+
+@args (String s)
+@switch (s) {
+case("test")   { this is a test}
```

Comment:
Does the ( parenthese next to the case keyword need to be directly next to it to work? In most of the rocker docs, we generally allow whitepsace in similar situations for if/else blocks.  Same feedback on the "default", whitespace before the next { bracket.

So I'd suggest you add just a couple more use cases in these examples with whitespace for "case ("val")" and maybe another unit test demonstrating "default { }" with whitespace also works.

Also, some examples of using keywords IN the html blocks would be good to verify as well.  So inside a "case" block, output the word "case" as html to make sure the parser doesn't break.  I'd also throw the keyword "default" into the html blocks to validate those don't break either.

Reply:
will add more test cases , as for whitespaces it should be fine (but ill verify).
regarding whitespace inside the switch this caused an issue (since it will generate invalid code) the way i solved this was by removing any whitespace (and failing for non whitespace) at switch exit:

see  exitSwitchBlock calling verifySwitchBlock

Reply:
@jjlauer done , LMK if this is what you meant ( see rocker/parser/SwitchBlockWithDefaultSpaces.rocker.html and  testSwitchBlockWithDefaultSpaces)